SparkPost/matchbox
# FE-772 Update Tag Component

### What Changed
- Restyles **Tag**

### How To Test or Verify
- Run storybook with `npm run start:storybook`
- Visit http://localhost:9001/?path=/story/feedback-tag--tag-with-remove
- Verify colors match mockups, using inspect mode
  - https://sparkpost.invisionapp.com/share/FJUVWXX7BXP#/393981944_Tag_Symbols
  - Tag with close button variations available in UI kit (ask Tim or Jon)

<img width="643" alt="Screen Shot 2020-02-03 at 9 25 45 PM" src="https://user-images.githubusercontent.com/3903325/73708284-eefbb480-46cb-11ea-9087-af358533f3bc.png">

### PR Checklist
